omim2obo/main.py

```diff
@@ -164,7 +164,7 @@ def omim2obo(use_cache: bool = False):
     # - Non-OMIM triples
     graph.add((URIRef('http://purl.obolibrary.org/obo/mondo/omim.owl'), RDF.type, OWL.Ontology))
     graph.add((URIRef(oboInOwl.hasSynonymType), RDF.type, OWL.AnnotationProperty))
-    graph.add((URIRef(MONDONS.omim_included), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.includedEntryInOMIM), RDF.type, OWL.AnnotationProperty))
```

Comment:
## Snake or camel case
Nico wrote:

> Look how clingen_label is formatted I think it's all upper case with underscore, so OMIM_INCLUDED should be as well (always check other examples of the same thing you are adding to determine correct casing)

Originally we were doing snake case, but then we 3 met a few weeks ago and coming out of that meeting, determined this and `omimFormerly` should be camel case, but I don't remember why.

@twhetzel Can make the judgement on this one?